steps/channel-create/step.yaml

```diff
@@ -24,9 +24,9 @@ examples:
     image: relaysh/slack-step-channel-create
     spec:
       connection: !Connection { type: slack, name: my-slack-account}
-      channel: "new-channel-name"
-      topic: "The New Hot Topic"
-      memberID: "UDHPDS88Z"
+      channel: !Parameter channel-name # Example: "relay-notifications"
+      topic: !Parameter channel-topic # Example: "Notifications for Relay workflows"
+      memberID: !Parameter memberID # Example: "UDHPDS88Z"
```

Comment:
FYI, comments like this won't currently be rendered when the example is shown to users. It's OK if you intend it for internal documentation however.

Reply:
Ok sounds good.